plugins/ocp/ocp-nvme.h

```diff
@@ -17,9 +17,9 @@ PLUGIN(NAME("ocp", "OCP cloud SSD extensions", NVME_VERSION),
 	COMMAND_LIST(
 		ENTRY("smart-add-log", "Retrieve extended SMART Information", ocp_smart_add_log)
 		ENTRY("latency-monitor-log", "Get Latency Monitor Log Page",
-		      ocp_latency_monitor_log)
+			ocp_latency_monitor_log)
 		ENTRY("clear-fw-activate-history", "Clear firmware update history log",
-		      clear_fw_update_history)
```

Comment:
This also same.

Reply:
@ikegami-t - The reason this is occurring is because tabs are exclusively being used. So function signatures with new lines are tabbing an extra level. Mixing tabs with whitespace seemed odd, but I'm not familiar with Linux code-style.

@igaw - Do you want me to change this, where it occurs? I'm happy to do so.

Reply:
I am pushing my changes, then you can do a followup if there are still obvious whitespaces damages

Reply:
Sounds good. I will make a note to align to @ikegami-t's comment on subsequent pull requests